linera-views/src/test_utils/performance.rs

```diff
@@ -220,8 +238,13 @@ where
 }
 
 /// Benchmarks the `write_batch` operation.
-pub async fn write_batch<S: TestKeyValueStore>(iterations: u64) -> Duration {
-    let store = S::new_test_store().await.unwrap();
+pub async fn write_batch<D>(iterations: u64) -> Duration
+where
+    D: TestKeyValueDatabase,
+    D::Store: KeyValueStore,
+{
+    let namespace = D::connect_test_namespace().await.unwrap();
+    let store = namespace.open_shared(&[]).unwrap();
```

Comment:
I think we could have `D::new_tesz_store(&[]).await.unwrap()` per the trait definition.

Reply:
good catch